genic/params.c

```diff
@@ -39,12 +39,14 @@ create_parameters()
     param_declare_double(ps, "w0_fld", OPTIONAL, -1., "Dark energy equation of state");
     param_declare_double(ps, "wa_fld", OPTIONAL, 0, "Dark energy evolution parameter");
 
+    param_declare_int(ps, "DifferentTransferFunctions", OPTIONAL, 0, "Use species specific transfer functions for baryon and CDM.");
```

Comment:
For the `MultipleTransferFunction` case, does it make more sense to have an abstraction of the primordial power rather than saving the transfer ratios? Because physically that's how the power spectrum of each species come about.

Reply:
It really makes more sense to save the transfer ratios, because the interpolation has to be able to deal with different k values for power and transfer functions.

Reply:
Aren't these just functions that return value of any give k -- I tend to wipe the data representation and interpolation algorithm under the carpet.You will need to interpolate both functions anyways because they don't match the mesh-k. But I think we can worry about these things later when the mood is right..